crates/cdk/src/generate.rs

```diff
@@ -0,0 +1,74 @@
+use std::{fmt::Debug, path::PathBuf};
+
+use anyhow::{Result, Error};
+
+use clap::Parser;
+use cargo_generate::{GenerateArgs, TemplatePath, generate};
+use include_dir::{Dir, include_dir};
+use tempfile::TempDir;
+
+static CONNECTOR_MODULE_TEMPLATE: Dir<'static> =
+    include_dir!("$CARGO_MANIFEST_DIR/../../connector/cargo_template");
+
+/// Generate new SmartModule project
```

Comment:
Need to replace all occurrences of `SmartModule` to `SmartConnector` in this file